daemon/changes.go

```diff
@@ -9,7 +9,9 @@ func (daemon *Daemon) ContainerChanges(job *engine.Job) engine.Status {
 		return job.Errorf("Usage: %s CONTAINER", job.Name)
 	}
 	name := job.Args[0]
-	if container := daemon.Get(name); container != nil {
+	if container, err := daemon.Get(name); err != nil {
+		return job.Error(err)
+	} else {
```

Comment:
Do you need this else here?  Seems unnecessary with the return in the `if` body.